vendor/vendor-ios/src/test/kotlin/com/malinskiy/marathon/ios/DerivedDataManagerTest.kt

```diff
@@ -106,7 +106,8 @@ class DerivedDataManagerTest {
         val privateKey = File(javaClass.classLoader.getResource("fixtures/derived-data-manager/test_rsa").file)
 
         @BeforeAll
-        fun `setup perimissions`() {
+        @JvmStatic
```

Comment:
This caused the failure for iOS tests due to wrong permissions on the private key